cms/templates/base.html

```diff
@@ -63,6 +63,7 @@
     <script type="text/javascript" src="${static.url(jsi18n_path)}"></script>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="path_prefix" content="${EDX_ROOT_URL}">
+    <%block name="header_meta"></%block>
```

Comment:
Not sure if I should include this one at the moment. It is good to have but it serve no purpose at the moment

Reply:
I think it's a good change